e2e/tests/test_e2e.py

```diff
@@ -212,6 +220,61 @@ def test_logical_backup_cron_job(self):
         self.assertEqual(0, len(jobs),
                          "Expected 0 logical backup jobs, found {}".format(len(jobs)))
 
+    @timeout_decorator.timeout(TEST_TIMEOUT_SEC)
+    def test_node_selector(self):
+        k8s = self.k8s
+
+        # 1. Add a label to a node
+        target_node = k8s.api.core_v1.read_node(name='postgres-operator-e2e-tests-worker2')
```

Comment:
I would rather see the current master node been labeled here, and not hard-coded by name (see taint test)